protected/humhub/modules/post/permissions/CreatePost.php

```diff
@@ -34,7 +34,6 @@ class CreatePost extends BasePermission
      * @inheritdoc
      */
     protected $fixedGroups = [
-        Space::USERGROUP_USER,
```

Comment:
Wouldn't it be better to instead of removing this line just move to to `$defaultAllowedGroups` array?

Reply:
Sure, I'll try that. Not sure what the implications are, either way, but that seems more logical.

Reply:
Okay, I tested it on my development instance, and indeed that works equally well. I've made the change, squashed, and force-pushed.

Reply:
Oh, I just realized that moving it to this `$defaultAllowedGroups` array means that the default for the CreatePost option will be set to `Allow`, of course! So is that what we want? It might be unexpected behavior for users.

Reply:
Simply removing it and not adding it to the aforementioned array keeps the default setting as `Deny`, but it just makes the option visible.

Reply:
Thoughts on this point @ArchBlood @luke- ?